drivers/mtd/Kconfig

```diff
@@ -86,6 +86,12 @@ config MODULE_MTD_MCI
     depends on CPU_FAM_LPC23XX
     select MODULE_MCI
 
+config MODULE_MTD_SDCARD_DEFAULT
+    bool "Use Generic SD card configuration"
+    depends on MODULE_MTD_SDCARD
```

Comment:
Or should this be `select MODULE_MTD_SDCARD` or `imply MODULE_MTD_SDCARD`?

Or add a `default y if MODULE_MTD_SDCARD_DEFAULT` to `MODULE_MTD_SDCARD`?

Reply:
Yeah, modelling dependencies in Kconfig is sometimes a nightmare. I always struggle when I have created the dependencies in the makefiles and try map them into Kconfig. This is often because the logic behind differs. While the makefiles pull in everything that is needed, the Kconfig often requires that the user selects something first in `menuconfig` before other dependencies can be resolved. Let me check.

Reply:
`drivers/mtd_sdcard/Kconfig` should define the feature `HAVE_` and the `MODULE_`.